docs/app/assets/css/docs_old.css

```diff
@@ -0,0 +1,696 @@
+html, body {
```

Comment:
This file is a copy of the old `docs.css` (duh...), which was left here for easy reference (until the new styles are finalized); so technically this is not needed (especially if the intention is move more toward `angular.io` styling.layout).